packages/api/taskOrderFiles/createTaskOrderFile.ts

```diff
@@ -0,0 +1,66 @@
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
+import {v4 as uuid} from "uuid";
+import {ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode} from "../utils/response";
+import {PutObjectCommand, S3Client} from "@aws-sdk/client-s3";
+import {FileMetadata, FileScanStatus} from "../models/FileMetadata";
+import * as parser from "lambda-multipart-parser"
+import {ErrorCodes} from "../models/Error";
+
+const bucketName = process.env.PENDING_BUCKET;
+
+/**
+ * Creates a new Task Order File
+ *
+ * @param event - The POST request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+    const key = uuid();
+    const client = new S3Client({});
```

Comment:
Eventually we may need client configs (for endpoints and such). We're going to need to move this to another function at some point. DynamoDB will need the same. Not a request. Just identifying something we need to plan for.